### PR TITLE
Simplify function doc samples

### PR DESCRIPTION
**Pull request by grant**
- Shorthand `=== undefined`
- Use maps rather than switch statements

### CODE REVIEW

File: functions/http/index.js [hunk 1]
```diff
@@ -23,14 +23,14 @@
  * @param {Object} res ExpressJS object containing the HTTP response to send.
  */
 exports.helloWorld = (req, res) => {
-  if (req.body.message === undefined) {
+  if (req.body.message) {
```
Comment (ace-n) [inline functions/http/index.js]: Missing an `!`?

File: functions/http/index.js [hunk 2]
```diff
@@ -44,41 +44,28 @@ exports.helloWorld = (req, res) => {
...
 exports.helloContent = (req, res) => {
-  let name;
-
-  switch (req.get('content-type')) {
-    // '{"name":"John"}'
-    case 'application/json':
-      name = req.body.name;
-      break;
-
+  let name = {
```
Comment (ace-n) [inline functions/http/index.js]: @jmdobry @MylesBorins I like this pattern - is it sufficiently idiomatic?

File: functions/http/index.js [hunk 3]
```diff
@@ -44,41 +44,28 @@ exports.helloWorld = (req, res) => {
...
+    'application/x-www-form-urlencoded': req.body.name;
+  }[req.get('content-type')];
 
   res.status(200).send(`Hello ${name || 'World'}!`);
 };
 // [END functions_http_content]
 
 // [START functions_http_method]
-function handleGET (req, res) {
+function handleGET(req, res) {
```
Comment (ace-n) [inline functions/http/index.js]: Please revert this. This is enforced by our linter (`semistandard`).

File: functions/http/index.js (hunk 1 above)
Comment (grant) [reply functions/http/index.js]: Done.

**Comment by grant** [comment]
I'll have to fix the lint errors.

**Pull request closed without merging**